test/integration/scheduler/predicates_test.go

```diff
@@ -52,11 +55,18 @@ func TestInterPodAffinity(t *testing.T) {
 	podLabel := map[string]string{"service": "securityscan"}
 	podLabel2 := map[string]string{"security": "S1"}
 
+	if err := createNamespacesWithLabels(cs, []string{"ns1", "ns2"}, map[string]string{"team": "team1"}); err != nil {
+		t.Fatal(err)
+	}
+	if err := createNamespacesWithLabels(cs, []string{"ns3"}, map[string]string{"team": "team2"}); err != nil {
+		t.Fatal(err)
+	}
+	defaultNS := "ns1"
```

Comment:
As we've moved the test with namespaceSelector as a separate test, we can revert the logic to use default namespace as it was.

Reply:
We need those changes because the feature is now beta and the plugin makes a namespace look up for the incoming pod namespace. It will work without it as we assume empty label set by default, but that will cause the following error message: https://github.com/kubernetes/kubernetes/blob/21ab9c417562223bfe3329830788bc05ee29fb3f/pkg/scheduler/framework/plugins/interpodaffinity/plugin.go#L137

Reply:
Ah, framework.CreateTestingNamespace() doesn't really create a namesapce object:

https://github.com/kubernetes/kubernetes/blob/9a67558784354c09487a980f1dac8ca76cd1ce90/test/integration/framework/util.go#L49

Here we need to create namespaces explicitly via API.